librz/bin/bin.c

```diff
@@ -1239,19 +1239,17 @@ RZ_API RZ_OWN RzList *rz_bin_section_flag_to_list(RzBin *bin, ut64 flag) {
 }
 
 RZ_API RzBinFile *rz_bin_file_at(RzBin *bin, ut64 at) {
-	RzListIter *it, *it2;
+	RzListIter *it;
 	RzBinFile *bf;
-	RzBinSection *s;
 	rz_list_foreach (bin->binfiles, it, bf) {
-		// chk for baddr + size of no section is covering anything
-		// we should honor maps not sections imho
-		rz_list_foreach (bf->o->sections, it2, s) {
-			if (at >= s->vaddr && at < (s->vaddr + s->vsize)) {
+		if (bf->o) {
```

Comment:
Reverse the condition and call `continue`?

Reply:
i will, it is just wip for now as i wanted to see which tests failed